src/test/java/app/studnicki/ox/ScoreTest.java

```diff
@@ -0,0 +1,58 @@
+package app.studnicki.ox;
```

Comment:
This class should be in same package as implementation class. Pls move it to app.studnicki.ox.game.
After this move you can change access your implementation class to package-private